### PR TITLE
Update sugargame and fix arrow keys not working

### PR DESCRIPTION
**Pull request by vaibhav-sangwan**
* Update sugargame
* Update SDL2 keycodes for UP, DOWN, LEFT and RIGHT arrow keys

I am not sure what keycode mapping the first elements of tuples on L13:L20 of g.py represented, probably [GLFW](https://www.glfw.org/docs/3.3/group__keys.html)?

@chimosky @walterbender @quozl please review

### CODE REVIEW

File: g.py [hunk 1]
```diff
@@ -10,10 +10,10 @@
...
 # only one 100% per game
 
-UP=(264,273)
-DOWN=(258,274)
-LEFT=(260,276)
-RIGHT=(262,275)
+UP=(264, 1073741906)
+DOWN=(258, 1073741905)
+LEFT=(260, 1073741904)
+RIGHT=(262, 1073741903)
```
Comment (chimosky) [inline g.py]: Haven't looked at the commit that introduced this, but it'll be better not to have hard coded values for things like this as they're bound to change.

Would be great to drop the hard coded values.

**Comment by chimosky** [comment]
Reviewed c697a2b, not tested.

**Comment by vaibhav-sangwan** [comment]
I've replaced the hardcoded literals. However, I was left with very little choice of key events to incorporate following actions:
* Start new game
* Put letter under cursor on active cell
* Show/hide help

So, I assigned K_TAB, K_LSHIFT and K_RSHIFT to these actions, which is very unconventional. Do you have any better suggestions?

**Comment by quozl** [comment]
Please replace like with like, and keep the feature of multiple keys to do the same function.  For example, on python2 pygame 1.9.1, 273 is K_UP, 264 is K_KP8.  You've changed it to ignore K_KP8.  Don't do that.  Teachers or students may have local language lesson plans that mention both.  Please use the pygame source for the date this code was originally written.  It was around version 1.9.1.  Tags are present in the upstream repository https://github.com/pygame/pygame

Or, these may be SDL constants that Pygame is exposing.  The SDL release close to when this activity was written was 1.2.14.

You've committed a game state data file.  Don't do that.  Pay close attention to what is changed, and only add to a commit what you changed.

**Comment by vaibhav-sangwan** [comment]
Yes, these were SDLv1.2 constants that pygame was exposing. A full list of the mappings is present [here](https://github.com/libsdl-org/SDL-1.2/blob/f5f2d3825637169fc8f3ebc6ef0d076916e27e82/include/SDL_keysym.h#L238). 

I've made the required changes. I've replaced the integer constants with pygame constants, while keeping the feature of multiple keys. Had to force push the changes as the changes made in [bf609b7](https://github.com/sugarlabs/across-and-down-activity/commit/bf609b79c2a9686e697a0242d779687ecb62fd16) weren't required.

> You've committed a game state data file. Don't do that. Pay close attention to what is changed, and only add to a commit what you changed.

Oh, I must have missed it while making the commit. I've fixed it now and will make sure to be more careful in the future.

**Pull request merged**